sql/ts/src/skdb_database.ts

```diff
@@ -13,6 +13,10 @@ import { SKDBTable } from "./skdb_util.js";
 import { SKDBGroupImpl } from "./skdb_group.js";
 import { connect } from "./skdb_orchestration.js";
 
+type DiffAcc = {
+  [key: string]: { since: bigint };
+};
```

Comment:
AFAIU this needs to be a named type definition. I'd be happy if anyone knows the syntax to inline this into the one occurrence.

Reply:
Does `const acc: { [key: string]: { since: bigint } } = {};` not work? What is the error?

Reply:
You're right, it works, my failure was a missing set of braces. :-/